README.md

```diff
@@ -116,6 +118,11 @@ for testing new thresholds before they come into affect on production.
 
 ## Setup
 
+Please install the following dependencies prior to running the application setup:
+
+brew install shared-mime-info
+brew install postgresql
```

Comment:
Fair point - most developers would have installed these as Apply which is generally the first service we use :+1: